Lib/_pyrepl/console.py

```diff
@@ -185,7 +191,12 @@ def runsource(self, source, filename="<input>", symbol="single"):
             the_symbol = symbol if stmt is last_stmt else "exec"
             item = wrapper([stmt])
             try:
-                code = self.compile.compiler(item, filename, the_symbol, dont_inherit=True)
+                code = self.compile.compiler(
+                    item,
+                    filename,
+                    the_symbol,
+                    dont_inherit=True,
```

Comment:
If we're modifying this anyway, we could remove this unused keyword:

```suggestion
```

Reply:
Good catch!